cli/npm/registry.rs

```diff
@@ -95,6 +97,14 @@ impl CliNpmRegistryApi {
     self.inner().get_cached_package_info(name)
   }
 
+  // TODO(nayeemrmn): Maybe declare in `NpmRegistryApi`.
```

Comment:
```suggestion
```

The `NpmRegistryApi` trait in `deno_npm` shouldn't have this function because the trait should only have functionality that `deno_npm` needs and we don't want to require implementors to implement this when it's not used there.